grapple/types/images.py

```diff
@@ -71,64 +106,75 @@ class ImageObjectType(DjangoObjectType):
     sizes = graphene.String(required=True)
     collection = graphene.Field(lambda: CollectionObjectType, required=True)
     tags = graphene.List(graphene.NonNull(lambda: TagObjectType), required=True)
-    rendition = graphene.Field(
-        lambda: get_rendition_type(),
-        max=graphene.String(),
-        min=graphene.String(),
-        width=graphene.Int(),
-        height=graphene.Int(),
-        fill=graphene.String(),
-        format=graphene.String(),
-        bgcolor=graphene.String(),
-        jpegquality=graphene.Int(),
-        webpquality=graphene.Int(),
-    )
+    rendition = graphene.Field(get_rendition_type, **get_rendition_field_kwargs())
     src_set = graphene.String(
         sizes=graphene.List(graphene.Int), format=graphene.String()
     )
+    if WAGTAIL_VERSION > (5, 0):
+        is_svg = graphene.Boolean(required=True)
 
     class Meta:
         model = WagtailImage
 
-    def resolve_rendition(instance, info, **kwargs):
+    def resolve_rendition(
+        instance: WagtailImage, info: GraphQLResolveInfo, **kwargs
+    ) -> WagtailImageRendition | None:
         """
         Render a custom rendition of the current image.
         """
-        filters = "|".join([f"{key}-{val}" for key, val in kwargs.items()])
+        preserve_svg = kwargs.pop("preserve_svg", False)
+        filter_specs = "|".join([f"{key}-{val}" for key, val in kwargs.items()])
 
         # Only allowed the defined filters (thus renditions)
-        if not rendition_allowed(filters):
+        if not rendition_allowed(filter_specs):
+            return
```

Comment:
It would be great to return specific exceptions here (and elsewhere) that can be handled by the client instead of returning nothing.